src/main/utilities/headers/Polygon.hpp

```diff
@@ -27,8 +28,8 @@ class Polygon {
     vector<vector<GLfloat>> vertices;  // 2D vector for vertices
     vector<vector<GLfloat>> textureCoords;  // 2D vector for texture coord data
     vector<vector<GLfloat>> normalCoords;  // 2D vector for normal coord data
-    GLuint textureUniformId;  // ID for finding texture sampler in OpenGL table
-    vector<GLint> pointCount;  // no. of distinct points in shape
+    GLint textureUniformId;  // ID for finding texture sampler in OpenGL table
```

Comment:
See comment above. VBOs should NOT be stored in the Polygon class. This is a bit of a mess; we'll need to spend some time sorting this out later.